scripts/main.js

```diff
@@ -65,6 +66,17 @@ $updateAppForm.on('submit', function (event) {
       })
     }
 })
+$deletebtn.on('click',function(event){
+	event.preventDefault()
+	var id = $('#deletebtn').data('id')
+	console.log(id)
+	applist.find(id)
+	.then (function(app){
+		applist.remove(id)
+		//renderAppList ()
+		setRoute('')
+	})
+})
```

Comment:
use spaces (2 space) instead of tabs for indentation. You should be able to change that setting in your editor